src/components/PoolOverview.tsx

```diff
@@ -153,9 +153,15 @@ export default function PoolOverview({
               <div className="margin Apr" key={symbol}>
                 {symbol.includes("/") ? (
                   <span className="label underline">
-                    <ToolTip content={symbol.replaceAll("/", "\n")}>
-                      Reward APR
-                    </ToolTip>
+                    <Tooltip
+                      title={
+                        <React.Fragment>
```

Comment:
I believe this doesn't require React.Fragment.

Reply:
You are right!